carbon-design-system/carbon
# fix(bug): #15291 wrong selectedItems count

Closes #15291 

the selected count is wrong after click the keyboard button  "space" or "enter" 

#### Changelog

**New**

- Only the specific item is focused , the keyboard enevt will responsed otherwise ignore it.

![image](https://github.com/carbon-design-system/carbon/assets/18441947/f3abe9fd-124e-4041-8cba-25207bf1d560)




#### Testing / Reviewing
Please go to the storybook for testing
https://react.carbondesignsystem.com/?path=/docs/components-multiselect--overview.
 - Open a multiselect
 - Open a multiselect
 - Select at least one item
 - Start pressing space or enter (no effect)
 - Press up/down button to focus the specific item, then  "space" or "enter" button will work as expected.
